src/test/java/org/broadinstitute/hellbender/utils/pileup/PileupElementUnitTest.java

```diff
@@ -155,6 +157,8 @@ public void testPrevAndNextTest(final GATKRead read, final CigarOperator firstOp
         Assert.assertEquals(pe2.getBetweenPrevPosition().size(), ops.size());
         Assert.assertEquals(pe2.getBetweenNextPosition().size(), 0);
         assertEqualsOperators(pe2.getBetweenPrevPosition(), ops);
+        Assert.assertEquals(pe2.getNeighborNotOnGenomeOperator(PileupElement.Direction.PREV), ops.get(ops.size()-1));
+        Assert.assertEquals(pe2.getNeighborNotOnGenomeOperator(PileupElement.Direction.NEXT), null);
```

Comment:
These tests for the new `getNeighborNotOnGenomeOperator()` method do not really seem adequate -- can we get a high-quality standalone unit test that properly exercises the method?